packages/web-components/src/image/image.stories.ts

```diff
@@ -0,0 +1,245 @@
+import { html } from '@microsoft/fast-element';
+import type { Args, Meta } from '@storybook/html';
+import { renderComponent } from '../__test__/helpers.js';
+import type { Image as FluentImage } from './image.js';
+import { BorderRadius, ImageFit, ImageShape } from './image.options.js';
+import './define.js';
+
+type ImageStoryArgs = Args & FluentImage;
+type ImageStoryMeta = Meta<ImageStoryArgs>;
+
+const imageTemplate = html<ImageStoryArgs>`
+  <div style="padding: 48px 24px; background-color: rgb(250, 250, 250);">
+    <fluent-image
+      ?block=${x => x.block}
+      ?border=${x => x.border}
+      borderRadius=${x => x.borderRadius}
```

Comment:
I'd update this to match the attribute (instead of the property) as it's an attribute on the element.
```suggestion
      border-radius=${x => x.borderRadius}
```